src/rust/engine/fs/fs_util/src/main.rs

```diff
@@ -216,7 +216,7 @@ fn execute(top_match: &clap::ArgMatches) -> Result<(), ExitError> {
             cas_address.to_owned(),
             1,
             chunk_size,
-            Duration::from_secs(30),
+            Duration::from_secs(30 * 60),
```

Comment:
All of the PR description probably could have gone here in a comment instead. For next time!